fuel/app/themes/default/partials/catalog/qset.php

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="qsetHistoryCtrl">
+	
+	<form id="import-form">
+		<h1>Previous Saves</h1>
+		<table id="qset-table" width="100%">
+			<thead width="100%">
+				<tr>
+					<th>Save Count</th>
+					<th>Saved At</th>
+				</tr>
+			</thead>
+			<tr ng-repeat="save in saves" ng-click="loadSaveData(save.id)">
+				<td>Save #{{saves.length - $index}}</td>
+				<td>{{save.created_at}}</td>
+			</tr>
+		</table>
+	</form>
+	<div class="no-saves" ng-show="saves.length == 0">
```

Comment:
I think most of the legacy styles are understore based? prefer to match if that's true.